dsls/sql/src/main/java/org/apache/beam/dsls/sql/rel/BeamJoinRel.java

```diff
@@ -242,17 +266,22 @@ private BeamSqlRow buildNullRow(BeamRelNode relNode) {
     return nullRow;
   }
 
-  private List<Pair<Integer, Integer>> extractJoinColumns(int separator) {
+  private List<Pair<Integer, Integer>> extractJoinColumns(int leftRowColumnCount) {
+    // it's a CROSS JOIN because: condition == true
+    if (condition instanceof RexLiteral && (Boolean) ((RexLiteral) condition).getValue()) {
```

Comment:
For my own edification, can you explain this a little further? What is the condition here, and why does it being set to true imply CROSS JOIN?